source/settings.rst

```diff
@@ -447,677 +445,785 @@ depending on whether or not they are distributed with that repository.
 
-    .. setting:: Script Context
+.. settingsgroup:: Commit
 
-      .. setting:: Show in RevisionGrid
+  .. setting:: Push forced with lease when Commit & Push action is performed with Amend option checked
 
-        If checked, the script is added to the context menu that is displayed when right-clicking on a line in the Revision Graph page.
+    In the Commit dialog, users can commit and push changes with one click. However, if changes are meant to amend
+    an already pushed commit, a standard push action will be rejected by the remote server. If this option is
+    enabled, a push action with ``--force-with-lease`` switch will be performed instead. The ``--force-with-lease``
+    switch will be added only when the ``Amend`` option is checked.
 
-  .. settingspage:: Hotkeys
+.. settingsgroup:: Updates
 
-    This page allows you to define keyboard shortcuts to actions when specific pages of Git Extensions are displayed.
-    The HotKeyable Items identifies a page within Git Extensions. Selecting a Hotkeyable Item displays the list of
-    commands on that page that can have a hotkey associated with them.
+  .. setting:: Check for updates weekly
 
-    The Hotkeyable Items consist of the following pages
+    Check for newer version every week.
 
-    1) Commit: the page displayed when a Commit is requested via the ``Commit`` User Menu button or the ``Commands/Commit`` menu option.
-    2) Browse: the Revision Graph page (the page displayed after a repository is selected from the dashboard (Start Page)).
-    3) RevisionGrid: the list of commits in Browse and other forms.
-    4) FileViewer: the page displayed when viewing the contents of a file.
-    5) FormMergeConflicts: the page displayed when merge conflicts are detected that need correcting.
-    6) BrowseDiff: Diff tab in Browse.
-    7) RevisionFileTree: The FileTree tab in Browse.
-    8) Scripts: shows scripts defined in Git Extensions and allows shortcuts to be assigned. Refer :ref:`settings-git-extensions-scripts`.
+  .. setting:: Check for release candidate versions
 
-    .. setting:: Hotkey
+    Include release candidate versions when checking for a newer version.
 
-      After selecting a Hotkeyable Item and the Command, the current keyboard shortcut associated with the command is displayed here.
-      To alter this shortcut, click in the box where the current hotkey is shown and press the new keyboard combination.
+.. settingspage:: Confirmations
 
-      .. settingbutton:: Apply
+This page allows you to turn off certain confirmation popup windows.
 
-        Click to apply the new keyboard combination to the currently selected Command.
+.. settingsgroup:: Confirm actions
 
-      .. settingbutton:: Clear
+  .. settingsgroup:: Commits
 
-        Sets the keyboard shortcut for the currently selected Command to 'None'.
+  .. setting:: Amend last commit
 
-    .. settingbutton:: Reset all Hotkeys to defaults
+    If checked, do not display the popup warning about
```

Comment:
These settings have been inverted.
```suggestion
    If not checked, do not display the popup warning about
```
or
```suggestion
    If checked, display the popup warning about
```

Reply:
ok

Reply:
This applies to all confirmation settings.

Reply:
Updating all